src/test/java/org/broadinstitute/hellbender/utils/haplotype/HaplotypeBAMWriterUnitTest.java

```diff
@@ -10,6 +10,7 @@
 import htsjdk.samtools.util.Locatable;
 import htsjdk.samtools.SamFiles;
 
+import java.nio.file.Path;
```

Comment:
You should add a test (either here or in `HaplotypeCallerIntegrationTest` which asserts that the tool can actually write into the test bucket (similar to the `ApplyBQSRIntegrationTest.testApplyBQSRCloud()` test you wrote for a one of your other branches.

Reply:
OK, I added it to `HaplotypeCallerIntegrationTest`, in the "bucket" group.